metricbeat/docs/autodiscover-aws-ec2-config.asciidoc

```diff
@@ -0,0 +1,25 @@
+{beatname_uc} supports templates for modules:
+
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+metricbeat.autodiscover:
+  providers:
+    - type: aws_ec2
+      period: 1m
+      credential_profile_name: elastic-beats
+      templates:
+        - condition:
+            equals:
+              meta.ec2.tags.service: "mysql"
```

Comment:
this is outdated, it should be `meta.aws.ec2..`. After the suggested changes (meta is somehow hidden), it would be `aws.ec2.tags.service`

Reply:
😂 I updated the PR description but forgot to update the actual doc... Will update this right away. Thanks!